src/frontend/src/style.scss

```diff
@@ -141,15 +141,39 @@ $account-label-padding-top: 20px;
 $account-label-font-size: 20px;
 
 // TRANSACTION HISTORY
-$received-color: rgba(235, 255, 204, 0.75);
-$reward-color: rgba(255, 248, 149, 0.75); 
-$sent-color:  rgba(255, 206, 192, 0.75);
-$no-transactions-text-color: gray;
+$receive-background-color: rgba(0, 128, 0, 0.356);
+$receive-color:rgb(197, 255, 197);
+
+$send-background-color: rgba(255, 0, 0, 0.294);
+$send-color:rgb(255, 198, 198);
+
+$reward-background-color: rgba(255, 255, 0, 0.336);
+$reward-color:rgb(255, 255, 202);
 
-$content-text-padding: 15px;
-$content-text-margin: -2px;
+$no-transactions-text-color: gray;
 
 $content-padding: 20px;
-$content-background-color: rgb(231, 231, 231);
+$content-background-color: black;
 $content-border-radius: 20px;
-$content-font-size: 20px;
+$content-height: 650px;
+
+
+$type-text-font-weight: 300;
+$type-text-font-size: 14px;
+
+$address-text-font-weight: bolder;
+$address-text-font-size: 18px;
+
+$row-amount-border-radius: 10px;
+$row-amount-padding: 5px;
+$row-amount-margin-top: 5%;
+
+$list-row-border-bottom: 1px solid white;
+$list-row-color: white;
+$list-row-background-color: black;
+$list-row-padding: 20px;
+
+$transaction-history-back-button-margin-top: 25px;
+$transaction-history-header-font-weight: bold;
```

Comment:
Maybe create a default font weight that is bold and can be used wherever we bold the font

Reply:
Now that you point it out it's kind've redundant to do this as "bold" is already an abstracted value. I'll just use bold in the component instead